app/plugin_hooks_test.go

```diff
@@ -754,13 +731,9 @@ func TestUserWillLogInIn_Passed(t *testing.T) {
 	w := httptest.NewRecorder()
 	session, err := th.App.DoLogin(w, r, th.BasicUser, "")
 
-	if err != nil {
-		t.Errorf("Expected nil, got %s", err)
-	}
+	assert.Empty(t, err, "Expected nil, got %s", err)
 
-	if session.UserId != th.BasicUser.Id {
-		t.Errorf("Expected %s, got %s", th.BasicUser.Id, session.UserId)
-	}
+	assert.Equal(t, session.UserId, th.BasicUser.Id)
```

Comment:
This assertion is missing the error message

Reply:
I don't think is needed, the `assert.Equal` is going to give you that exact information anyway.

Reply:
Totally true, resolving